admin_manual/configuration_user/user_auth_ldap.rst

```diff
@@ -2,6 +2,8 @@
 User Authentication with LDAP
 =============================
 
+.. warning:: Please check advanced and expert configuration carefully before starting production use.
```

Comment:
I'd suggest this: "Please check both the advanced and expert configurations carefully before using in production.". I'm assuming that you're meaning to be careful before deploying to a production environment?